events/iwslt2022.md

```diff
@@ -0,0 +1,47 @@
+---
+parent: Events
+title: IWSLT 2022
+description: International Conference on Spoken Language Translation (IWSLT)
+location: Dublin, Ireland
+name: IWSLT 2022
+startDate: "2022-05-22"
+endDate: "2022-05-27"
+---
+
+The International Conference on Spoken Language Translation (**IWSLT**) is an annual scientific conference on spoken language translation.
+
+IWSLT 2022 will be held in ACL 2022 online and in Dublin, Ireland on 26-27 May 2022.
+
+*Last updated from https://iwslt.org/2022/ on January 12th, 2022*
+
+The event will host shared tasks:
+
+- Simultaneous Speech Translation (Chairs: Juan Pino, Meta; Katsuhito Sudoh, NAIST)
```

Comment:
IMHO parens are almost always a sign of bad formatting.

I prefer something like this:

> - **Simultaneous Speech Translation**
>   - Chairs: 
>      - Juan Pino, Meta
>      - Katsuhito Sudoh, NAIST

or just 

> - **Simultaneous Speech Translation**
>    - Juan Pino, Meta
>    - Katsuhito Sudoh, NAIST